CMakeLists.txt

```diff
@@ -96,6 +97,20 @@ if(CLR_CMAKE_PLATFORM_UNIX)
         -Wno-switch # switch values not handled
         -Wno-int-to-pointer-cast
     )
+
+    set(cxx_compile_flags
+        -fdelayed-template-parsing
+    )
+
+    add_compile_options(
+        -fms-extensions
+        -msse4.1
+        "$<$<COMPILE_LANGUAGE:C>:${c_warn_flags}>"
+        "$<$<COMPILE_LANGUAGE:CXX>:${cxx_warn_flags}>"
+        "$<$<COMPILE_LANGUAGE:CXX>:${cxx_compile_flags}>"
+    )
+
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fdelayed-template-parsing")
```

Comment:
`-fdelayed-template-parsing` also specified in `cxx_compile_flags/add_compile_options`?